test/gtest/ucp/ucp_test.cc

```diff
@@ -604,6 +604,13 @@ bool ucp_test::check_tls(const std::string& tls)
     return cache[tls] = (status == UCS_OK);
 }
 
+void ucp_test::skip_external_protov1() const
+{
+    if (!m_ucp_config->ctx.proto_enable) {
+        UCS_TEST_SKIP_R("skip forced UCX_PROTO_ENABLE=n");
+    }
+}
```

Comment:
IMO we should not need this function since we didn't skip external req with protov1 so far, so no need to skip it after this PR as well